### PR TITLE
Fix example in the docs

### PR DESCRIPTION
**Pull request by voronaam**
Documentation contained invalid YAML syntax - extra commas. I removed them.

Not related to any library.

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [na] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [na] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by prince-chrismc — DISMISSED** [review]

File: docs/package_templates/meson_package/all/conandata.yml [hunk 1]
```diff
@@ -2,12 +2,12 @@ sources:
...
-    - "https://mirror1.net/package-1.2.0.tar.gz",
-    - "https://mirror2.net/package-1.2.0.tar.gz",
+    - "https://mirror1.net/package-1.2.0.tar.gz"
+    - "https://mirror2.net/package-1.2.0.tar.gz"
     sha256: "________________________________________________________________"
   "1.1.0":
-    - "https://mirror1.net/package-1.1.0.tar.gz",
-    - "https://mirror2.net/package-1.1.0.tar.gz",
+    - "https://mirror1.net/package-1.1.0.tar.gz"
+    - "https://mirror2.net/package-1.1.0.tar.gz"
```
Comment (SpaceIm) [inline docs/package_templates/meson_package/all/conandata.yml:10]: url is missing

Comment (voronaam) [reply docs/package_templates/meson_package/all/conandata.yml]: Indeed, I missed this extra mistake. Fixed it. Note that this is a template file in the docs. Still, I feel it should show a correct example.

**Comment by prince-chrismc** [comment]
They are probably all wrong 🙈

**Review by SSE4 — DISMISSED** [review]

**Review by RubenRBS — APPROVED** [review]
Thanks a lot :)

**Comment by prince-chrismc** [comment]
Make sure to comment on #4

**Review by SSE4 — DISMISSED** [review]

**Comment by prince-chrismc** [comment]
Please do not force push 🙏 GitHub forces us to restart the review which is not fun!

**Review by prince-chrismc — APPROVED** [review]

**Comment by voronaam** [comment]
> Please do not force push pray GitHub forces us to restart the review which is not fun!

Sorry. I was using the "Update branch" button on the PR to keep this one up to date. But since I was choosing "rebase" variant of it, it was ending with a force push. I will not do that again.

**Comment by prince-chrismc** [comment]
No need to update the branch, once a build passes we just need the boot to come back :)

**Pull request merged**